<launch>
	<param name="config_file" value="$(find lightswarm_core)/params/config.yaml" />

	<!-- run april tag node with kinect params -->
	<include file="$(find april_tag)/launch/april_tag_kinect.launch" />
	<!-- run april tag based detector with stub everything -->
	<node name="april_detector" pkg="lightswarm_sense" type="april_detector.py"/>

	<node name="stub_simulator" pkg="lightswarm_core" type="stub_simulator.py"/>
    <node name="shadow_calculator" pkg="lightswarm_shadow" type="shadow_calculator.py"/>
    <node name="renderer" pkg="lightswarm_render" type="renderer.py"/>

</launch>